app/data/0_star_wars.json

```diff
@@ -127,6 +127,11 @@
                                 }
                             }
 
+                        },
+                        {
+                            "goto": {
+                                "id": "96682325-47ab-41e4-a56e-8315a19ffe2a"
+                            }
```

Comment:
Do we need to specify a `goto` route if it's just going to the next block? Assuming the previous routing rules don't fire, doesn't it just route to the next block anyway?